tokio-quiche/src/socket/capabilities.rs

```diff
@@ -353,6 +392,7 @@ impl SocketCapabilities {
         #[cfg(feature = "perf-quic-listener-metrics")]
         let _ = b.rxtime();
         let _ = b.gro();
+        let _ = b.rcvmark();
```

Comment:
do we need to handle the error if it doesn't get applied? Maybe metrics in the future?